server/services/database/users.js

```diff
@@ -52,14 +64,27 @@ const createUser = ({ email, password, name, role }) => {
               console.log(error);
               reject("An unexpected error occured, please try again later.");
             }
-
             resolve(result.rows);
           }
         );
-      });
+      })
+        .then(() => getUserIdByEmail(email))
+        .then(id => submitUserLanguages(id.id, languages));
     });
 };
 
+const submitUserLanguages = (id, languages) => {
+  const values = languages
+    .map((language, index) => `($1,$${2 + index})`)
+    .join(", ");
+  const sqlQuery = `INSERT INTO users_languages (user_id, language_code) VALUES ${values}`;
+
+  return pool
+    .query(sqlQuery, [id, ...languages])
+    .then(result => result.rows)
+    .catch(error => console.error(error));
```

Comment:
You don't need to catch the error here, because if you have an error here, it will just console.log it. Instead, this error should be caught in the route code and return an error to the frontend.